src/visitors/rules/registry/registryDependencies.js

```diff
@@ -6,37 +6,30 @@ class RegsitryDependencies {
     return 'registry-dependencies';
   }
 
-  OpenAPIRoot() {
-    return {
-      onEnter: async (_node, _definition, ctx) => {
-        this.redoclyClient = new RedoclyClient();
-        if (this.redoclyClient.hasToken()) {
-          ctx.headers = [...ctx.headers, {
-            matches: `https://api.${process.env.REDOCLY_DOMAIN || 'redoc.ly'}/registry/**`,
-            name: 'Authorization',
-            value: (this.redoclyClient && await this.redoclyClient.getAuthorizationHeader()) || '',
-          }];
-        }
-      },
-      onExit: async (_node, _definition, ctx) => {
-        if (process.env.UPDATE_REGISTRY && this.redoclyClient.hasToken()) {
-          await this.redoclyClient.updateDependencies(ctx.registryDependencies);
-        }
-      },
-    };
+  async OpenAPIRoot(_node, _definition, ctx) {
+    this.redoclyClient = new RedoclyClient();
+    if (this.redoclyClient.hasToken()) {
+      ctx.headers = [...ctx.headers, {
+        matches: `https://api.${process.env.REDOCLY_DOMAIN || 'redoc.ly'}/registry/**`,
+        name: 'Authorization',
+        value: (this.redoclyClient && await this.redoclyClient.getAuthorizationHeader()) || '',
+      }];
+    }
   }
 
-  any() {
-    return {
-      onEnter: (_node, _definition, ctx, unresolvedNode) => {
-        if (unresolvedNode.$ref) {
-          const link = unresolvedNode.$ref.split('#/')[0];
-          if (isFullyQualifiedUrl(link) && RedoclyClient.isRegistryURL(link)) {
-            ctx.registryDependencies.push(link);
-          }
-        }
-      },
-    };
+  async OpenAPIRoot_exit(_node, _definition, ctx) {
+    if (process.env.UPDATE_REGISTRY && this.redoclyClient.hasToken()) {
+      await this.redoclyClient.updateDependencies(ctx.registryDependencies);
+    }
+  }
+
+  enter(_node, _definition, ctx, unresolvedNode) {
+    if (unresolvedNode.$ref) {
+      const link = unresolvedNode.$ref.split('#/')[0];
+      if (isFullyQualifiedUrl(link) && RedoclyClient.isRegistryURL(link)) {
```

Comment:
```suggestion
      if (isFullyQualifiedUrl(link) && RedoclyClient.isRegistryUrl(link)) {
```
Notice inconsistent use of `URL` `Url`. I prefer `Url` when camelCase.